### PR TITLE
feat: /aws redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied changes as per [doc](https://docs.google.com/document/d/1lkH-MSPmgjCMP4gI0FW29YIKVDvBuddxA63iQhmsYIU/edit) and [design](https://www.figma.com/design/FoWXkNxxdri9EgSsLdtTrH/24.10-AWS-bubble?node-id=0-1&m=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-14223.demos.haus/aws
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12837

### CODE REVIEW

File: templates/aws/index.html [hunk 1]
```diff
@@ -1,160 +1,285 @@
...
-              height="288",
-              hi_def=True,
-              loading="lazy",
-              attrs={"class": "p-logo-section__logo"}
-              ) | safe
-            }}
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="u-no-margin--bottom" />
```
Comment (lizzochek) [inline templates/aws/index.html:54]: makes sense to use a p-rule here

**Review by lizzochek — APPROVED** [review]

**Comment by lizzochek** [comment]
please fix failing checks before merging

File: templates/aws/index.html (hunk 1 above)
Comment (mtruj013) [reply templates/aws/index.html:54]: `u-no-margin--bottom` is the replacement for `p-rule`, pr [here](https://github.com/canonical/vanilla-framework/pull/5241/files)

**Comment by mattea-turic** [comment]
@mtruj013 

Image on right should be 600x400 (3:2), think it;s 16:9 rn
Screenshot 2024-08-23 at 14 33 56

Not sure how easy this is to implement, but are you able to have the ctas from both pricing cops (i.e. ubuntu and ubuntu pro for aws) line up? Although it's not something that's a big deal so only if it's a quick fix
Screenshot 2024-08-23 at 14 39 31

**Comment by mtruj013** [comment]
Design +1 given over MM, merging

**Pull request merged**